evaluation/Aggregation.py

```diff
@@ -4,36 +4,37 @@
 import numpy as np
 
 class Aggregation:
-    def __init__(self, epsilon=1.0, t = 1):
+    def __init__(self, epsilon=1.0, t = 1, repeatcount = 10000):
```

Comment:
python convention has no spaces for named variables

t=1, repeat_count=10000 etc.

Reply:
Removed spaces